### PR TITLE
Update java customer registry sample.

### PR DESCRIPTION
**Pull request by RayRoestenburg**
Refs #285

### CODE REVIEW

File: samples/java-valueentity-customer-registry/src/main/java/customer/Main.java [hunk 1]
```diff
@@ -27,6 +27,18 @@ public static AkkaServerless createAkkaServerless() {
...
+            .registerView(
+                    CustomerViewModel.getDescriptor().findServiceByName("CustomerSummaryByName"),
+                    "customerSummaryByName",
+                    CustomerDomain.getDescriptor())
+            .registerView(
+                    CustomerViewModel.getDescriptor().findServiceByName("CustomersResponseByName"),
+                    "customersResponseByName",
+                    CustomerDomain.getDescriptor())
             .register(CustomerValueEntityProvider.of(CustomerValueEntity::new))
             .register(CustomerActionProvider.of(CustomerActionImpl::new));
```
Comment (patriknw) [inline samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:43]: we could change this to `AkkaServerlessFactory.withComponents`

File: samples/java-valueentity-customer-registry/src/main/java/customer/Main.java [hunk 2]
```diff
@@ -27,6 +27,18 @@ public static AkkaServerless createAkkaServerless() {
                     "customerByName",
                     CustomerDomain.getDescriptor())
             // end::register[]
+            .registerView(
```
Comment (patriknw) [inline samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:30]: `registerView` must be some leftover, it should be `.register(ViewProvider)`. Can you remove the old `registerView` and fix any references.

Comment (johanandren) [reply samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:30]: In the eventing sample I found it easiest to update main by renaming it to something else (but retaining the Main in pom.xml) and regenerating it with codegen and then copying over the annotations.

Comment (RayRoestenburg) [reply samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:30]: Thanks, will do.

File: samples/java-valueentity-customer-registry/src/main/java/customer/Main.java (hunk 1 above)
Comment (RayRoestenburg) [reply samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:43]: I'll have a look, thanks.

Comment (RayRoestenburg) [reply samples/java-valueentity-customer-registry/src/main/java/customer/Main.java]: Done.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
@RayRoestenburg Can you also remove this obsolete method: https://github.com/lightbend/akkaserverless-java-sdk/blob/55267e0d7dd51d690bea8e5a7092167deb05fe92/sdk/src/main/java/com/akkaserverless/javasdk/AkkaServerless.java#L211

either in this PR or in a separate

**Review by johanandren — APPROVED** [review]

File: samples/java-valueentity-customer-registry/src/main/java/customer/view/CustomerByEmailView.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* This code is managed by Akka Serverless tooling.
+ * It will be re-generated to reflect any changes to your protobuf definitions.
+ * DO NOT EDIT
```
Comment (raboof) [inline samples/java-valueentity-customer-registry/src/main/java/customer/view/CustomerByEmailView.java:3]: Ah, that's not right, added #291

**Review by raboof — APPROVED** [review]
LGTM, but replace the comments with the 'right' one

**Pull request merged**